markets/bfp-market/contracts/storage/Margin.sol

```diff
@@ -382,6 +409,15 @@ library Margin {
 
         // Ensure that there is collateralUsd on the account to ensure this account margin can be liquidated.
         Margin.MarginValues memory marginValues = Margin.getMarginUsd(accountId, market, price);
-        return marginValues.discountedMarginUsd == 0 && marginValues.collateralUsd != 0;
+
+        return
+            marginValues.discountedMarginUsd.toInt() -
+                getMarginLiquidationOnlyReward(
```

Comment:
As mentioned this deduction of `getMarginLiquidationOnlyReward` also needs to be included in `modifyCollateral`

Reply:
Yep thanks! Addressed